client/src/view/welcome.tsx

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright (c) 2019. Beautiful Code BV, Rotterdam, Netherlands
+ * Licensed under GNU GENERAL PUBLIC LICENSE Version 3.
+ */
+
+import * as React from "react"
+import { Button, Col, Container, Row } from "reactstrap"
+import { Subject } from "rxjs"
+
+import { FabricKernel } from "../body/fabric-kernel"
+import { Island } from "../island/island"
+import { IAppState, logString } from "../state/app-state"
+import { Transition } from "../state/transition"
+import { IStorage } from "../storage/storage"
+
+const GITHUB = "https://github.com/beautiful-code-bv/galapagotchi"
+const GITTER = "https://gitter.im/galapagotchis/community"
+const GMAIL = "galapagotchi@gmail.com"
+const TWITTER = "https://twitter.com/galapagotchi"
+const WEBASSEMBLY = "https://webassembly.org/"
+const WEBGL = "https://en.wikipedia.org/wiki/WebGL"
+const THREEJS = "https://threejs.org/"
+const DARWINS_DANGEROUS_IDEA = "https://en.wikipedia.org/wiki/Darwin%27s_Dangerous_Idea"
+
+interface IRef {
+    href: string
+    text: string
+}
+
+function Ref(props: IRef): JSX.Element {
+    return <span> <a href={props.href} target="_blank">{props.text}</a> </span>
+}
+
+export interface IWelcomeProps {
+    userId?: string
+    storage: IStorage
+    fabricKernel: FabricKernel
+    stateSubject: Subject<IAppState>
+    ownedLots: string[]
+}
+
+export interface IWelcomeState {
+    developer?: boolean
+}
+
+export class Welcome extends React.Component<IWelcomeProps, IWelcomeState> {
+
+    constructor(props: IWelcomeProps) {
+        super(props)
+        this.state = {developer: false}
+    }
+
+    public render(): JSX.Element {
+        return (
+            <div className="welcome">{this.page}</div>
+        )
+    }
+
+    private get page(): JSX.Element {
+        if (this.props.userId) {
+            return this.welcomeBack
+        }
+        if (this.state.developer) {
+            return this.welcomeDeveloper
+        }
+        return this.welcomeN00b
+    }
+
+    // N00b ============================================================================================================
+
+    private get welcomeN00b(): JSX.Element {
+        return (
+            <div>
+                <h3>Welcome to the Galapagotchi project!</h3>
+                <img src="logo.png"/>
+                <p>
+                    It appears that this is your first visit, at least with this web browser.
+                </p>
+                <p>
+                    This project aims to give you a fascinating experience.
```

Comment:
Good statement, but experience of what?

Reply:
This project aims to give you a fascinating experience that shows you how things can evolve by Darwin's natural selection.